docs/feature-flags/feature-flags-logs.md

```diff
@@ -38,7 +38,11 @@ The table below indicates which attributes should be added to the
 [LogRecord](https://github.com/open-telemetry/opentelemetry-specification/tree/v1.31.0/specification/logs/data-model.md#log-and-event-record-definition) and their types.
 
 <!-- semconv log-feature_flag -->
-The event name MUST be `feature_flag`
```

Comment:
I think this sentence should be moved out of semconv generation, 1 line up

Reply:
It used to be auto-generated. I can add it back in, actually.

Reply:
Are they? where it comes from?

Reply:
This is now fixed.